pkg/descheduler/pod/pods.go

```diff
@@ -97,7 +99,21 @@ func ListPodsOnANode(client clientset.Interface, node *v1.Node) ([]*v1.Pod, erro
 }
 
 func IsCriticalPod(pod *v1.Pod) bool {
-	return types.IsCriticalPod(pod)
+	return IsCritical(pod.Namespace, pod.Annotations)
+}
+
+func IsCritical(ns string, annotations map[string]string) bool {
+	// Critical pods are restricted to "kube-system" and "openshift-*" namespaces as of now.
+	if ns != api.NamespaceSystem {
```

Comment:
Probably adding some context helps as well, something like PR openshift/origin#19104 is merged because of which we shouldn't evict pods from "openshift-*"